### DISCUSSION TITLE
Ports not starting with new version on runpod

### DISCUSSION THREAD

**Discussion by jairavula** [ashleykleynhans/stable-diffusion-docker#67] [General]
Hello,

Is anyone else having issue with the new version (6.8.3) of the docker image on runpod? The container starts but most of the services are not available except for the jupyter notebook, which seems like it is missing most directories. Also this version takes much longer to build the pod. Is there a way to use a previous version in the meantime?

**Comment by ashleykleynhans** [comment]
There is nothing wrong with the new version, you have to wait for it to finish syncing.
